src/components/sbb-tab-title/sbb-tab-title.scss

```diff
@@ -0,0 +1,118 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  display: inline-block;
+  max-width: 100%;
+  -webkit-tap-highlight-color: transparent;
+}
+
+.sbb-tab-title {
+  position: relative;
+  margin: 0;
+  min-height: var(--sbb-spacing-fixed-10x);
+  display: flex;
+  align-items: center;
+  padding-block: var(--sbb-spacing-fixed-2x);
+  padding-inline: var(--sbb-spacing-fixed-5x);
+  gap: var(--sbb-spacing-fixed-1x);
+  user-select: none;
+  cursor: pointer;
+  transition-duration: var(--sbb-animation-duration-4x);
+  transition-timing-function: var(--sbb-animation-easing);
+  transition-property: color;
+
+  &::before {
+    position: absolute;
+    content: '';
+    inset: 0;
+    border: var(--sbb-border-width-1x) solid var(--sbb-color-cloud-default);
+    border-radius: var(--sbb-spacing-fixed-6x);
```

Comment:
looks a little bit weird to use this token. maybe use just 1em? or a very big value. When it should be exact, then it should be half of the title size (20px in this case).